src/api/public/apidocs-new/paths/source_project_cmd_createkey.yaml

```
@@ -0,0 +1,45 @@
+post:
+  summary: Generate a new gpg key for a project
```

```suggestion
  summary: Generate a new gpg key for a project.
```
Missing dot at the end of the sentence.

---

It's updated.